function-support.html

```diff
@@ -1214,14 +1225,43 @@ <h2 id="logic">Logic and Set Theory</h2>
     </div>
 
 <h2 id="macros">Macros</h2>
+	<table class="symbol">
+		<tr>
+			<td>\(\def\foo{x^2}\foo + \foo\)</td>
+			<td><strong>\def</strong>\foo{x^2}&nbsp;\foo + \foo</td>
+		</tr>
+		<tr>
+			<td>\(\gdef\bar#1{#1^2}&nbsp;\bar{y} + \bar{y}\)</td>
+			<td><strong>\gdef</strong>\bar#1{#1^2}&nbsp;\bar{y} + \bar{y}</td>
+		</tr>
+		<tr> <td></td>  <td>\global\def\macroname[args]{definition}</td> </tr>

```

Comment:
I find this one a bit confusing, given that `[` actually means `[` in the examples below.  Perhaps replace `[args]` with `...` or `#1#2...`?